packages/expo-mail-composer/ios/EXMailComposer/EXMailComposer.m

```diff
@@ -35,7 +35,7 @@ - (dispatch_queue_t)methodQueue
                     rejecter:(UMPromiseRejectBlock)reject)
 {
   if (![MFMailComposeViewController canSendMail]) {
-    reject(@"E_COMPOSE_UNAVAILABLE", @"Mail services are not available.", nil);
+    reject(@"E_COMPOSE_UNAVAILABLE", @"Mail services are not available. Make sure you're signed into the Mail app", nil);
```

Comment:
Good catch! Could you also make this change in `ios/versioned-react-native/ABI35_0_0/…/EXMailComposer.m`, so this change lands in SDK35 too (not only on `master` and SDK36+)?

Reply:
```suggestion
    reject(@"E_COMPOSE_UNAVAILABLE", @"Mail services are not available. Make sure you're signed into the Mail app", nil);
    resolve(@{ @"status": @"failed" });
```